lib/pymedphys/tests/mosaiq/test_session_logic.py

```diff
@@ -106,22 +120,23 @@ def test_session_offsets_for_site(
 
         # for each treatment field
         previous_session_number = None
-        previous_session_offset_when = None
 
         for session_number, session_offset in session_offsets_for_site(
             connection, sit_set_id
         ):
-            if previous_session_number is not None:
+            if previous_session_number:
                 # check that the sessions are in order
                 assert session_number > previous_session_number
 
-            if session_offset is not None:
-                if previous_session_offset_when is not None:
-                    assert session_offset[0] > previous_session_offset_when
-                assert abs(session_offset[1]) <= 10.0
-                assert abs(session_offset[2]) <= 10.0
-                assert abs(session_offset[3]) <= 10.0
+            if np.any(session_offset):
+                assert np.max(np.abs(session_offset)) <= 10.0
+
+            mean_session_offset = mean_session_offset_for_site(connection, sit_set_id)
+            assert np.any(mean_session_offset)
+            assert np.max(np.abs(mean_session_offset)) <= 10.0
```

Comment:
Would we be able to have another test here that creates a static, non-random, well defined set of offsets for a new site ID and just verifies that those exact numbers are pulled correctly with `localization_offset_for_site` and that `mean_session_offset_for_site` appropriately returns the mean as expected?